src/MiniProfiler.Shared/MiniProfiler.Shared.csproj

```diff
@@ -22,6 +22,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="NETStandard.Library" Version="1.6.1" />
+    <PackageReference Include="System.Collections.Immutable" Version="1.3.0" />
```

Comment:
Oops, i should remove it, it is a left over from my experiments with using immutable collections